rails/client/app/bundles/kaiju/components/Project/containers/StoryBoardContainer.jsx

```diff
@@ -7,7 +7,9 @@ import Card from './CardContainer';
 
 const mapStateToProps = ({ project, workspaces }) => ({
   project,
-  workspaces: Object.keys(workspaces).map(key => <Card key={key} id={key} />),
+  workspaces: Object.keys(workspaces).sort((a, b) => (
+    workspaces[a].name > workspaces[b].name
+  )).map(key => <Card key={key} id={key} />),
```

Comment:
These are sorted on the client because the user can dynamically add and remove workspaces on the client. The array refreshed directly from the server response, but instead remains in sync with the actions.